config/core/test-configs.yaml

```diff
@@ -2515,6 +2535,7 @@ test_configs:
     test_plans:
       - baseline
       - cros-ec
+      - kselftest-alsa
```

Comment:
There are issues with mainline support for this platform, mainly some bits missing in the arm64 defconfig. So we could leave this here but it might not work for a little while until that's resolved. Most likely, we'll have to add things to the `arm64-chromebook` fragment in KernelCI if the changes aren't accepted upstream.

Reply:
OK: https://staging.kernelci.org/test/plan/id/625944fba216fa032205f531/